static/js/browser2/browser.ts

```diff
@@ -34,11 +34,15 @@ window.onload = () => {
   if (!_.isEmpty(statVarId)) {
     document.title = `${statVarId} - ${document.title}`;
   }
-  axios
+  const typesPromise = axios
     .get(`/api/browser/propvals/typeOf/${dcid}`)
-    .then((resp) => {
-      const values = resp.data.values;
-      const types = values.out ? values.out : [];
+    .then((resp) => resp.data);
+  const statVarsPromise = axios
+    .get(`/api/place/statsvars/${dcid}`)
```

Comment:
although we should not make a new endpoint in the mixer, we should make a new endpoint in flask for this (don't make the client load all that data just for a boolean -- do this on the server)

Reply:
done